index.html

```diff
@@ -1816,7 +1816,7 @@ <h5>
             <p its-locale-filter-list="zh-hant" lang="zh-hant">當行尾出現結束夾注符號時，可以縮減其末側二分之一個漢字大小的空白。</p>
           </li>
           <li id="id104">
-            <p its-locale-filter-list="en" lang="en">According to section 5.1.10 of the <cite>General Rules for Punctuation</cite> (GB/T 15834―2011), when one character wide punctuation appears at the end of a line, half a character space can be reduced behind it.</p>
+            <p its-locale-filter-list="en" lang="en">According to section 5.1.10 of the General Rules for Punctuation (GB/T 15834-2011), when a full-width punctuation character appears at the end of a line, it can be reduced to half a character width.</p>
```

Comment:
`cite` element seems to be missing?

Reply:
@xfq ops, it's a copy mistake. I have recovered it, thx~